account_move_name_sequence/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 
 {
     "name": "Account Move Number Sequence",
-    "version": "16.0.1.1.3",
```

Comment:
There are commits in the branch 16.0 that are missing here, for instance a015ec61f8e432a96376c236ea48df0106e29a5e

Version before migration should be 16.0.1.1.4